pythonquize42.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "130239a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "cb2f34c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def getsalary_record():\n",
+    "    sid=int(input(\"enter id\"))\n",
+    "    sname=input(\"enter name\")\n",
+    "    salary=int(input(\"enter salary\"))\n",
+    "    return[sid,sname,salary]"
```

Comment:
is space require between return and list?

Reply:
no require space between return and list